marklogic-data-hub-central/ui/e2e/cypress/e2e/explore/exploreUnmerge.cy.tsx

```diff
@@ -250,7 +250,9 @@ describe("Test '/Explore' graph right panel", () => {
     browsePage.getUnmergeIcon().first().scrollIntoView().click();
     compareValuesModal.getModal().should("be.visible");
     compareValuesModal.getInclusionCheckbox().click();
-    compareValuesModal.getUnmergeButton().click();
+    compareValuesModal.getUnmergeButton().click().then(() => {
+      cy.findByText("Are you sure you want to unmerge this document? Doing so, the original documents will be moved out of the archive collection and also considered for future matches.").should("be.visible");
```

Comment:
I think the text here is missing one word compared to the one in the ticket description. "**By** doing so, the original documents...etc."

Reply:
oops, adding it!